appinfo/info.xml

```diff
@@ -23,7 +23,7 @@
 	<screenshot>https://github.com/nextcloud/approval/raw/main/img/screenshot_2.jpg</screenshot>
 	<screenshot>https://github.com/nextcloud/approval/raw/main/img/screenshot_3.jpg</screenshot>
 	<dependencies>
-		<nextcloud min-version="22" max-version="27"/>
```

Comment:
Should we adjust min version to 24 then, if we no longer test on < 24?

Reply:
Yes why not. Done. Thanks